soh/src/overlays/actors/ovl_En_Go2/z_en_go2.c

```diff
@@ -1063,24 +1059,28 @@ void EnGo2_BiggoronSetTextId(EnGo2* this, GlobalContext* globalCtx, Player* play
     u16 textId;
 
     if ((this->actor.params & 0x1F) == GORON_DMT_BIGGORON) {
-        if (gSaveContext.bgsFlag) {
+        if ((!gSaveContext.n64ddFlag && gSaveContext.bgsFlag)) {
```

Comment:
Very small nitpick. The other conditionals in this file have these 2 the other way around.

Reply:
Fine with me 👍